proxy/manager.go

```diff
@@ -33,10 +42,17 @@ func (proxy *Proxy) manage(ctx context.Context, w http.ResponseWriter, req *http
 	}
 
 	defer func() {
-		if closeErr := babbageResponse.Body.Close(); closeErr != nil {
+		if closeErr := clientResponse.Body.Close(); closeErr != nil {
 			log.Error(ctx, "error closing the response body", closeErr)
 		}
 	}()
 
-	response.WriteResponse(ctx, w, babbageResponse, req, cfg)
+	response.WriteResponse(ctx, w, clientResponse, req, cfg)
+}
+
+func IsReleaseCalendarURL(url string) bool {
+	if strings.Contains(url, "/releases/") || strings.Contains(url, "/releasecalendar") {
```

Comment:
This can be simplified to `return strings.Contains(url, "/releases/") || strings.Contains(url, "/releasecalendar")`.

However, are you sure this logic is correct? This will be looking for those two substrings in any part of the URL, but in reality we want them to be at the very beginning, don't we? I could be wrong, though.